pinax/submissions/compat.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from account.decorators import LoginRequired
```

Comment:
LoginRequired is not used by this package. login_required is imported in views.py